kernel/obj_core.c

```diff
@@ -112,3 +115,236 @@ int k_obj_type_walk_unlocked(struct k_obj_type *type,
 
 	return status;
 }
+
+#ifdef CONFIG_OBJ_CORE_STATS
+int k_obj_stats_register(struct k_obj_core *obj_core, void *stats,
```

Comment:
I guess this and following functions are called `k_obj_core_stats...` instead of `k_obj_stats...`, from the `obj_core.h` above.

Reply:
Fixing.